src/main/java/com/fasterxml/jackson/databind/EnumNamingStrategy.java

```diff
@@ -0,0 +1,7 @@
+package com.fasterxml.jackson.databind;
+
+public interface EnumNamingStrategy {
+
+    public String translate(String value);
```

Comment:
Would be good to add Javadocs to explain direction, and name argument as something more specific than `value`.
I think that `value` would be `Enum.name()` to translate, and result is the "external" value used in JSON, right?